controls/include/controls/six_deg_PID.h

```diff
@@ -0,0 +1,44 @@
+//
+// Created by tanvi on 11/21/2020.
+//
+
+#ifndef CONTROLS_SIX_DEG_PID_H
+#define CONTROLS_SIX_DEG_PID_H
+#include "PID.h"
+#include <vector>
+
+class six_deg_PID
+{
+private:
+    PID roll_ = PID();
+    PID pitch_ = PID();
+    PID yaw_ = PID();
+
+    PID x_ = PID();

```

Comment:
I would rename x,y,z to surge,sway,heave. These are the terms commonly used in marine settings and underwater robotics. The reason it's helpful is x,y, and z may be defined differently on different vehicles. For instance, sometimes x is forward and sometimes y is forward. But, with surge, sway, heave, surge is always forward, sway is always lateral, and heave is always vertical.